src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -382,7 +382,10 @@ public string DefaultPythonEngine
         /// If enabled user's custom package locations will not be loaded.
         /// </summary>
         public bool DisableCustomPackageLocations { get; set; }
-
+        /// <summary>
+        /// If true the run mode will be set to Automatically, if false to Manual.
+        /// </summary>
+        public bool RunTypeAutomatic { get; set; }
```

Comment:
Can we serialize this info using string so we can just borrow `enum RunType { Manual, Automatic, Periodic }`? What would be the trade from your point of view?

Reply:
This variable is the one that saves the user's Default run setting, and being that the only options the user have are Manual or Automatic it doesn't feel right having the option to save Periodic on it too. On the other hand I believe using the enum as you suggested would make it more manageable in terms of code. I suppose the best option here depends on what we value the most, what I can say for sure is that the name of the property should be more representative of what it saves so I will change it on the next update to something more fitting

Reply:
@Astul-Betizagasti we can use `IsDefaultRuntypeAuto` etc, but that still looks less intuitive to me and less expandable. I prefer we serialize the actual type, even it is only going to either `Automatic` or `Manual` for now, but this allows us to expand in the future, e.g. when Dynamo become a web service, user could have time based run type etc.

Reply:
Make sense to me, I will update that